src/Maestro/Maestro.Web/Api/v2019_01_16/Models/Goal.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Maestro.Data.Migrations;
+using Maestro.Data.Models;
+
+namespace Maestro.Web.Api.v2019_01_16.Models
+{
+    public class Goal
+    {
+        public Goal([NotNull] Data.Models.GoalTime other)
+        {
+            if (other == null)
+            {
+                throw new ArgumentNullException(nameof(other));
+            }
+            DefinitionId = other.DefinitionId;
+            Minutes = other.Minutes;
+            ChannelId = other.ChannelId;
+        }
+
+        public int DefinitionId { get; set; }
+        public int ChannelId { get; set; }
+        public int Minutes { get; set; }
+        public class GoalData
```

Comment:
Why is there another class in here that also has Minutes in it? Nothing seems to use this.

Reply:
Oooh. It the request object.  I'd un-nest it can call it something like "GoalDataUpdateRequest" or something. :-)

Reply:
renamed class name